lms/static/scripts/frontend_apps/components/dashboard/DashboardApp.tsx

```diff
@@ -1,29 +1,60 @@
+import {
+  Link,
+  LogoIcon,
+  ProfileIcon,
+  SettingsIcon,
+} from '@hypothesis/frontend-shared';
 import classnames from 'classnames';
-import { Route, Switch, useParams } from 'wouter-preact';
+import { Link as RouterLink, Route, Switch, useParams } from 'wouter-preact';
 
+import { useConfig } from '../../config';
+import { urlPath } from '../../utils/api';
 import AssignmentActivity from './AssignmentActivity';
 import CourseActivity from './CourseActivity';
 import DashboardFooter from './DashboardFooter';
 import OrganizationActivity from './OrganizationActivity';
 
 export default function DashboardApp() {
+  const { user } = useConfig();
   const { organizationPublicId } = useParams<{
     organizationPublicId: string;
   }>();
 
   return (
     <div className="flex flex-col min-h-screen gap-5">
-      <div
-        className={classnames(
-          'flex justify-center p-3 w-full',
-          'bg-white border-b shadow',
-        )}
-      >
-        <img
-          alt="Hypothesis logo"
-          src="/static/images/hypothesis-wordmark-logo.png"
-          className="h-10"
-        />
+      <div className="px-3 py-4 bg-white border-b shadow">
+        <div className="flex justify-between items-center mx-auto max-w-6xl">
+          <RouterLink href={urlPath`/organization/${organizationPublicId}`}>
+            <LogoIcon />
+          </RouterLink>
+
+          <div className="flex gap-6">
+            {user && (
+              <span
+                className={classnames(
+                  'flex gap-2 items-center',
+                  'font-semibold text-color-text-light',
+                )}
+              >
+                <ProfileIcon />
+                {user.display_name}
+              </span>
```

Comment:
This thing is not a link, but it looks the same as the "Settings" item, which is a link. Only on hover you can see the difference.

I'm wondering if we should make them look different or this is enough, or even if the user should be a link to "somewhere".

Reply:
Common designs in other web properties are:

1. A single item in the header with a profile icon / name, which functions as an "account menu" toggle
2. An item in the header with profile icon / name which goes to a user's profile view, showing their timeline/activity etc., and a separate menu with links to account settings etc.

We don't have a "your activity" view yet, but this seems like a natural thing we might want to add in future.

Reply:
> We don't have a "your activity" view yet

This is the key point, I think. My question is more "what should we do until we have a place to link to?". Should we link to somewhere else in the interim, or just leave it as a non-link?

Reply:
Leave it as a non-link for the moment. We can change it later.